### ISSUE TITLE
what is the value in the ratio methods?

### ISSUE THREAD

**Issue by griii2** [brehaut/color-js#29]
Either lightenByRatio does not work or I don't understand what ratio to use. 

Great library btw!

**Comment by griii2** [comment]
I get it, it's a number from 0 to 1, there was a bug on my side of the code, sorry. Still the docs could mention this. Thanks for a great library!

**Comment by brehaut** [comment]
I agree. In hindsight I would have used a different word in the method names. I'll look at updating the docs in a few weeks (very busy currently); I'll happily accept a reasonable pull request for it though.
